force-app/main/default/lwc/courseRegistrationInformation/courseRegistrationInformation.js

```diff
@@ -0,0 +1,28 @@
+import { LightningElement, track } from 'lwc';
+
+export default class courseRegistrationInformation extends LightningElement {
+
+    @track Course__c;
```

Comment:
Du har kalt variabelen for Course__c her, men du bruker CourseId i html-en og på linje 11. Det er vel kanskje feil? Får det ikke til å fungere